src/backend/LDW.WebAPI/Controllers/v1/AccountController.cs

```diff
@@ -90,5 +91,26 @@ public async Task<IActionResult> ConfirmEmail([FromQuery] string email, string c
             }
             return Ok(Translations.VerifyEmailSuccess);
         }
+
+        [HttpPost]
+        [AllowAnonymous]
+        [Route("login")]
+        public async Task<IActionResult> Login([FromBody] LoginUserApiModel model)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState);
+            }
+
+            var isPasswordCorrect = await Mediator.Send(new GetUserLoginQuery(model.Email, model.Password));
+            if (!isPasswordCorrect)
+            {
+                return BadRequest(message: Translations.LOGIN_PASSWORD_INCORRECT);
```

Comment:
Will it return 400 status code? That's not very good (the request is actually correct, but credentials are not). It's better to use 401 Unauthorized: «*If the request already included Authorization credentials, then the 401 response indicates that authorization has been refused for those credentials.*»

Reply:
Updated return code to 401:)